addons/survey/wizard/survey_invite_views.xml

```diff
@@ -5,27 +5,30 @@
             <field name="name">survey.invite.view.form</field>
             <field name="model">survey.invite</field>
             <field name="arch" type="xml">
-                <form string="Compose Email" class="o_mail_composer_form">
+                <form string="Compose Email" class="o_mail_composer_form" style="padding-bottom: 0px;">
                     <group col="1">
                         <group col="2">
                             <field name="survey_access_mode" invisible="1"/>
                             <field name="survey_users_login_required" invisible="1"/>
                             <field name="survey_users_can_signup" invisible="1"/>
-                            <field name="survey_id" readonly="context.get('default_survey_id')"/>
+                            <field name="survey_id" readonly="context.get('default_survey_id')" invisible="1"/>
                             <field name="existing_mode" widget="radio" invisible="1" />
                             <field name="render_model" invisible="1"/>
-                            <field name="survey_start_url" label="Public share URL" readonly="1" widget="CopyClipboardChar"
+                            <field string="Survey Link" name="survey_start_url" label="Public share URL" readonly="1" widget="CopyClipboardChar"
                                  attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"
                                  class="mb16"/>
+                            <field string="Send by Email" name="shown" widget="boolean_toggle"
+                            attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"/>
```

Comment:
Do we really want to hide this button if the survey is not public ? isn't it something like: if public, display link + option to display email options, otherwise: don't display link and display composer ?